frontend/src/ClinicianVideo.tsx

```diff
@@ -119,6 +119,11 @@ class ClinicianVideoImpl extends React.Component<ClinicialVideoProps, ClinicalVi
         }
     }
     
+    async componentWillUnmount() {
+        const exportTranscript =  firebase.functions().httpsCallable('exportTranscript');
+        await exportTranscript({transcript: this.state.transcription, uid: this.props.user.uid, encounterId: this.props.encounterId});
```

Comment:
this.state.transcription is in a very browser specific format.  This is probably ok for the demo, but we need to think about moving the definition to the API side.